### DISCUSSION TITLE
song for  @teaprotocol  🍵

### DISCUSSION THREAD

**Discussion by chris-setra** [pkgxdev/pkgx#949] [Show and Tell]
https://x.com/xstsetra/status/1751534944806207640?s=20

![tea song bg](https://github.com/pkgxdev/pkgx/assets/116212442/e8033aa0-47e5-4934-ad4f-6e6f88fbe1d2)

the file mp4 is too big, i can't upload it here

**Comment by minhdataaa** [comment]
You can post in Social media channel (Tea's discord) 
ID: 1019339799712239726

**Reply by chris-setra** [reply]
All right
